### ISSUE TITLE
start.sh cannot run when called from outside its own directory

### ISSUE THREAD

**Issue by IllyaMoskvin** [docker/toolbox#687]
**TL;DR:** This is probably due to the relative `DOCKER_MACHINE` path definition in [start.sh#L10](DOCKER_MACHINE=./docker-machine.exe)

Basically, I installed Docker Toolbox and deselected the "VirtualBox" and "Git for Windows" options, since they were already installed... in _non-default locations_. This caused issues in a number or areas, mainly in that the "Docker Quickstart Terminal" shortcut refused to work (cf. #686). The path to `start.sh` was correct, but it assumed that Git was installed in its default location. It would try to launch it using `cmd`, and the terminal window would flash briefly, then exit, too fast for me to see the messages.

After running the Kitematic quick-start wizard thingy, I corrected the shortcut target to my Git Bash path:

```
C:\Tools\Git\git-bash.exe --login -i "C:\Tools\Docker\start.sh"
```

Alright, still flash-exiting. Next, I tried running it directly:

```
Administrator@DELL1545 MINGW64 ~
$ /c/Tools/Docker/start.sh
Docker Machine is not installed. Please re-run the Toolbox Installer and try again.
Looks like something went wrong in step ´Looking for vboxmanage.exe´...
Press any key to continue...
```

At least I'm not the only one having [this issue](https://stackoverflow.com/questions/39373217/docker-looks-something-went-wrong-in-step-looking-for-vboxmanage-exe). Doing this works:

```
Administrator@DELL1545 MINGW64 ~
$ cd /c/tools/docker

Administrator@DELL1545 MINGW64 /c/tools/docker
$ ./start.sh
```

I changed the shortcut target parameters to be a [command call](https://stackoverflow.com/questions/42849378/git-bash-shortcut-to-run-command):

```
C:\Tools\Git\git-bash.exe -c "cd /c/tools/docker; ./start.sh"
```

...which worked, but interestingly, this just flash-exits:

```
C:\Tools\Git\git-bash.exe --c "/c/tools/docker.sh"
```

Maybe the syntax is wrong? Eh. Anyway, `DOCKER_MACHINE` should not be relative. The script fails, because it cannot find `DOCKER_MACHINE` in the folder from which the script was called ([start.sh#L31](https://github.com/docker/toolbox/blob/7e92244d7497ecd432ed71ad5381a4c0ff56e02a/windows/start.sh#L31)). One solution would be to [get the directory where the script is located](https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within), and prepend it to `DOCKER_MACHINE` instead of using `./[...]`.

Also, I think the shortcut is fundamentally broken, but that might be a separate issue. I don't know enough about the `--login` and `-i` flags to say for sure.

**Comment by MaxPeal** [comment]
can you test if in you case #689 will help?
